tests/src/test/java/org/eclipse/hono/tests/registry/TenantAmqpIT.java

```diff
@@ -145,11 +180,16 @@ public void testGetConfiguredButNotCreatedTenantReturnsNotFound(final TestContex
     @Test
     public void testGetByCaSucceeds(final TestContext ctx) {
 
+        final String tenantId = Constants.DEFAULT_TENANT; //helper.getRandomTenantId();
```

Comment:
Should this be random tenant id?

Reply:
Nope, as it is tied to the hard coded values in the Auth service. See the discussion with @sophokles73 above :)

Reply:
OK. The comment is a bit confusing, as it implies it's on TODO. Maybe remove the comment and we'll change this in another pr

Reply:
i removed it